tv_apps/smart-system/js/crash_reporter.js

```diff
@@ -93,6 +96,7 @@ window.CrashReporter = (function() {
     };
 
     var button = null;
+    var systemBanner = new SystemBanner();
```

Comment:
Why do you want to move the creation of systemBanner to here? It looks the same and we don't need to do this change.